src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -350,12 +360,18 @@ const Replies = ({
     getAvatarUrl,
     getMentionWithQuery,
     getUserProfileUrl,
+    isParentPending = false,
     isRepliesLoading = false,
     mentionSelectorContacts,
+    onReplyCreate = noop,
```

Comment:
Will this function even make sense if the is a noop operation? Should we make onReplyCreate non optional?

Reply:
What if we don't want replies for a comment? They shouldn't need to add a cb for this then right?

Reply:
If we don't want to show replies then why even render the component?

Reply:
What do you mean? Shouldn't we have flexibility in having a BaseComment not have replies? Are you saying that all comments need to have the ability to create replies? We are also reusing BaseComment in the `Replies` component. That would mean we have to pass the create prop in each reply as well.